blueflood-elasticsearch/src/main/java/com/rackspacecloud/blueflood/io/AbstractElasticIO.java

```diff
@@ -142,83 +133,16 @@
         //token paths matching query, which also have a next level.
         tokenInfoBuilder.addTokenPathWithNextLevel(metricIndexData.getTokenPathsWithNextLevel());
 
-        Set<String> completeMetricsMatchingQueryPrevLevel = metricIndexData.getCompleteMetricNamesAtBaseLevel();
```

Comment:
We need to know if it is a complete metric name at base level to look for enums. If there are no enums we dont need it.

But I think we can adjust the regex expression a little bit, now that we dont need that information. We can create another story to look into this API after we cleanup enums.